mrtt-ui/src/styles/forms.js

```diff
@@ -0,0 +1,16 @@
+import { styled } from '@mui/material/styles'
+
+export const MainFormDiv = styled('div')(() => ({
```

Comment:
I think you can actually just use something like`styled.div` for standard HTML elements. 
Something like:
```js
export const MainFormDiv = styled.div(() => ({
  ...
}))
```
If you don't need props or a function, you can also use backticks instead:
```js
export const MainFormDiv = styled.div`
   ...
`
}))
```

Reply:
hmm I get errors if I try either of those 'Uncaught TypeError: _mui_material_styles__WEBPACK_IMPORTED_MODULE_0__.default.div is not a function`

Reply:
I am following this example from[ the docs](https://mui.com/system/styled/
<img width="575" alt="Screen Shot 2022-05-10 at 8 50 44 AM" src="https://user-images.githubusercontent.com/26089140/167632527-68e7d16f-cf4a-4130-81b0-f68910bdf21e.png">
)